openfisca_us/parameters/benefit/NSLP/benefit.yaml

```diff
@@ -0,0 +1,14 @@
+description: Annual value of NSLP per child
+
+2021-07-16: 595.8
+
+
+metadata:
+  unit: currency-USD
+  reference: 
+  - title: Federal Reserve Bank of Atlanta - Policy Rules Database
+    href: https://www.atlantafed.org/-/media/documents/economic-mobility-and-resilience/advancing-careers-for-low-income-families/policy-rules-database/policy-rules-database-technical-manual.pdf
+  reference:
```

Comment:
I think we just need to remove the `reference:` line here and this should be valid.